docs/api/devtime_dependencies.md

```
@@ -0,0 +1,275 @@
+
+## Comprehensive Examples
+
+### Example: Adding a development-time dependency
+
+This example demonstrates how to add a Redis store as a development-time dependency to the application, backed by the Testcontainers Redis Go module.
+
+```go
+package main
+
+import (
+	"context"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)